.travis.yml

```diff
@@ -21,6 +21,8 @@ matrix:
     - php: 7.1
     - php: 7.2
       env: DOCCHECK=true COVERAGE=true PHPUNIT_FLAGS="-v --coverage-text" VARNISH_MODULES_VERSION=0.12.1
```

Comment:
we only need dockcheck and coverage and varnish modules with one php version. lets use the newest. can you please remove this line so 7.2 builds with the defaults?